src/Admin/BaseMediaAdmin.php

```diff
@@ -63,7 +63,7 @@ public function prePersist(object $object): void
         $object->setContext($parameters['context']);
     }
 
-    public function configurePersistentParameters(): array
+    final public function configurePersistentParameters(): array
```

Comment:
not sure about this one.

Reply:
but this can be protected

Reply:
And prepersist can be protected too

Reply:
I will take a look at those protected, and change them in another Pr

Reply:
Revert the final keyword